action.yaml

```
@@ -29,6 +29,11 @@ runs:
       shell: bash
       working-directory: ${{ inputs.working_directory }}
       if: inputs.enable_aqua_install == 'true'
+    - run: |
+        echo "${AQUA_ROOT_DIR:-${XDG_DATA_HOME:-$HOME/.local/share}/aquaproj-aqua}/bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
```

this needs to be 
```suggestion
        echo "${AQUA_ROOT_DIR:-$env:LOCALAPPDATA/aquaproj-aqua}/bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
```

i don't think XDG is a thing on windows.